packages/client/src/hoc/withChecker.tsx

```diff
@@ -117,6 +118,10 @@ export default (Component: any) => (props: Props) => {
     return resetChecker();
   }
 
+  // @TODO We should handle this in routes.
+  if (!topic) {
+    return <NotFoundPage />;
+  }
```

Comment:
I think we don't need this logic. There is already an `if (topic) {` and in case we don't have a topic we load the page using a dynamic topic. What we need is an `error` in case the `fetch(` fails.